.msix install cannot find init script

### Code of Conduct

- [x] I agree to follow this project's Code of Conduct

### What happened?

So I tried installing using the new .msix installer on a new system. Starting OMP results in the following error:

```
PS C:\Users\myusername> oh-my-posh init pwsh | Invoke-Expression

& : The term 'C:\Users\myusername\AppData\Local\oh-my-posh\init.26.17.3.default.1072.ps1' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
At line:1 char:3
+ & 'C:\Users\myusername\AppData\Local\oh-my-posh\init.26.17.3.defaul ...
+   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (C:\Users\myuser...efault.1072.ps1:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException
```

The mentioned folder doesn't even exist:
`C:\Users\myusername\AppData\Local\oh-my-posh`

Upon searching, I only found the script in the following location:
`C:\Users\myusername\AppData\Local\Packages\ohmyposh.cli_96v55e8n804z4\LocalCache\Local\oh-my-posh\init.26.18.0.default.1072.ps1`

Running it manually and OMP starts appearing working fine:
`Invoke-Expression -Command "C:\Users\myusername\AppData\Local\Packages\ohmyposh.cli_96v55e8n804z4\LocalCache\Local\oh-my-posh\init.26.18.0.default.1072.ps1"`

### Theme

(irrelevant)

### What OS are you seeing the problem on?

Windows

### Which shell are you using?

powershell

### Log output

```Shell
PS C:\Users\myusername> oh-my-posh debug --plain

Version: 26.18.0

Shell: powershell

Prompt:

 myusername   ~                                                                                             in powershell at 15:51:49

Segments:

ConsoleTitle(true)                        -   0 ms
Session(true)                             -   0 ms
Path(true)                                -   4 ms
Git(false)                                -   4 ms
Root(false)                               -   0 ms
Status(true)                              -   1 ms
Node(false)                               -   0 ms
Go(false)                                 -   0 ms
Python(false)                             -   0 ms
Shell(true)                               -   0 ms
Time(true)                                -   0 ms

Run duration: 108.8784ms

Cache path: C:\Users\myusername\AppData\Local\oh-my-posh

Config path: no --config set, using default built-in configuration

Logs:

[DEBUG] 15:51:49.667 debug.go:35 → logging enabled, raw mode: true
[TRACE] 15:51:49.756 load.go:filePath() - 0s
[ERROR] 15:51:49.756 load.go:Load:38 → no config file specified
[TRACE] 15:51:49.756 load.go:Load() - 391.4µs
[TRACE] 15:51:49.756 path.go:Path() - 0s
[DEBUG] 15:51:49.756 file.go:Init:26 → loading cache file: C:\Users\myusername\AppData\Local\oh-my-posh\omp.cache
[DEBUG] 15:51:49.756 file.go:Init:47 → loading cache key: INITVERSIONPWSH
[TRACE] 15:51:49.756 file.go:Init(C:\Users\myusername\AppData\Local\oh-my-posh\omp.cache) - 755µs
[DEBUG] 15:51:49.756 file.go:Get:69 → cache is nil, returning empty value for key: prompt_count_cache
[TRACE] 15:51:49.756 terminal.go:setPromptCount() - 0s
[TRACE] 15:51:49.756 terminal.go:GOOS() - 0s
[DEBUG] 15:51:49.756 terminal.go:setPwd:115 → C:\Users\myusername
[TRACE] 15:51:49.756 terminal.go:setPwd() - 0s
[TRACE] 15:51:49.756 terminal.go:Init() - 755µs
[DEBUG] 15:51:49.756 terminal.go:Shell:368 → no shell name provided in flags, trying to detect it
[DEBUG] 15:51:49.765 terminal.go:Shell:379 → process name: powershell.exe
[TRACE] 15:51:49.765 terminal.go:Shell() - 8.5339ms
[TRACE] 15:51:49.765 terminal.go:Flags() - 0s
[DEBUG] 15:51:49.765 file.go:Get:69 → cache is nil, returning empty value for key: template_cache
[TRACE] 15:51:49.765 cache.go:restoreCache() - 0s
[TRACE] 15:51:49.765 terminal_windows.go:Root() - 0s
[TRACE] 15:51:49.765 terminal.go:Shell() - 0s
[TRACE] 15:51:49.765 terminal.go:Flags() - 0s
[TRACE] 15:51:49.765 terminal.go:StatusCodes() - 0s
[TRACE] 15:51:49.765 terminal_windows.go:IsWsl() - 0s
[TRACE] 15:51:49.765 terminal.go:Flags() - 0s
[TRACE] 15:51:49.765 terminal.go:Flags() - 0s
[DEBUG] 15:51:49.765 home.go:12 → C:\Users\myusername
[TRACE] 15:51:49.765 terminal_windows.go:IsWsl() - 0s
[TRACE] 15:51:49.765 terminal.go:Flags() - 0s
[TRACE] 15:51:49.765 terminal.go:Flags() - 0s
[TRACE] 15:51:49.765 terminal.go:GOOS() - 0s
[DEBUG] 15:51:49.765 terminal.go:User:249 → myusername
[TRACE] 15:51:49.765 terminal.go:User() - 0s
[DEBUG] 15:51:49.765 terminal.go:Host:266 → 8310HALSRVRDS15
[TRACE] 15:51:49.765 terminal.go:Host() - 0s
[TRACE] 15:51:49.765 terminal.go:GOOS() - 0s
[DEBUG] 15:51:49.765 terminal.go:Getenv:77 → NO DATA
[TRACE] 15:51:49.765 terminal.go:Getenv(SHLVL) - 0s
[DEBUG] 15:51:49.765 writer.go:Init:107 → terminal program: Windows Terminal
[DEBUG] 15:51:49.765 writer.go:Init:108 → terminal shell: shell
[DEBUG] 15:51:49.765 terminal.go:Getenv:77 → NO DATA
[TRACE] 15:51:49.765 terminal.go:Getenv(OMP_CACHE_DISABLED) - 0s
[DEBUG] 15:51:49.765 file.go:Get:69 → cache is nil, returning empty value for key: accent_color
[TRACE] 15:51:49.765 terminal_windows.go:WindowsRegistryKeyValue(HKEY_CURRENT_USER\Software\Microsoft\Windows\DWM\ColorizationColor) - 0s
[DEBUG] 15:51:49.765 terminal_windows.go:WindowsRegistryKeyValue:198 → ColorizationColor(DWORD): 0xC40078D7
[TRACE] 15:51:49.765 colors_windows.go:GetAccentColor() - 304.5µs
[TRACE] 15:51:49.765 colors.go:SetAccentColor() - 304.5µs
[TRACE] 15:51:49.765 terminal.go:Shell() - 0s
[DEBUG] 15:51:49.765 terminal.go:Getenv:77 → NO DATA
[TRACE] 15:51:49.765 terminal.go:Getenv(POSH_SHELL_VERSION) - 0s
[DEBUG] 15:51:49.765 debug.go:PrintDebug:24 → segment: Title
[DEBUG] 15:51:49.765 text.go:patchTemplate:149 → {{ .Shell }} in {{ .Folder }}
[TRACE] 15:51:49.766 terminal.go:Flags() - 0s
[DEBUG] 15:51:49.766 terminal.go:Getenv:77 → NO DATA
[TRACE] 15:51:49.766 terminal.go:Getenv(POSH_CURSOR_LINE) - 0s
[DEBUG] 15:51:49.766 terminal.go:Getenv:77 → NO DATA
[TRACE] 15:51:49.766 terminal.go:Getenv(POSH_CURSOR_COLUMN) - 0s
[TRACE] 15:51:49.766 terminal.go:Flags() - 0s
[TRACE] 15:51:49.766 terminal.go:Flags() - 0s
[TRACE] 15:51:49.766 terminal.go:Flags() - 0s
[DEBUG] 15:51:49.766 home.go:12 → C:\Users\myusername
[TRACE] 15:51:49.766 terminal.go:GOOS() - 0s
[DEBUG] 15:51:49.766 segment.go:Execute:108 → segment: Status
[DEBUG] 15:51:49.766 file.go:Get:69 → cache is nil, returning empty value for key: toggle_cache
[DEBUG] 15:51:49.766 segment.go:isToggled:229 → no toggles found
[TRACE] 15:51:49.767 terminal.go:StatusCodes() - 0s
[DEBUG] 15:51:49.767 properties.go:GetString:28 → status_template: {{ .Code }}
[TRACE] 15:51:49.767 terminal.go:Flags() - 0s
[DEBUG] 15:51:49.767 home.go:12 → C:\Users\myusername
[TRACE] 15:51:49.767 terminal.go:GOOS() - 0s
[DEBUG] 15:51:49.767 segment.go:Execute:108 → segment: Path
[DEBUG] 15:51:49.767 file.go:Get:69 → cache is nil, returning empty value for key: toggle_cache
[TRACE] 15:51:49.767 terminal.go:Flags() - 0s
[DEBUG] 15:51:49.767 segment.go:isToggled:229 → no toggles found
[DEBUG] 15:51:49.767 properties.go:GetBool:22 → display_cygpath: false
[TRACE] 15:51:49.767 terminal.go:GOOS() - 0s
[DEBUG] 15:51:49.767 home.go:12 → C:\Users\myusername
[TRACE] 15:51:49.767 terminal.go:GOOS() - 0s
[TRACE] 15:51:49.767 terminal.go:Shell() - 0s
[DEBUG] 15:51:49.767 segment.go:Execute:108 → segment: Session
[TRACE] 15:51:49.767 terminal.go:Shell() - 0s
[TRACE] 15:51:49.767 terminal.go:Flags() - 0s
[DEBUG] 15:51:49.767 file.go:Get:69 → cache is nil, returning empty value for key: toggle_cache
[DEBUG] 15:51:49.767 segment.go:isToggled:229 → no toggles found
[DEBUG] 15:51:49.767 terminal.go:Getenv:77 → NO DATA
[TRACE] 15:51:49.767 terminal.go:Getenv(SSH_CONNECTION) - 0s
[DEBUG] 15:51:49.767 terminal.go:Getenv:77 → NO DATA
[TRACE] 15:51:49.767 terminal.go:Getenv(SSH_CLIENT) - 0s
[TRACE] 15:51:49.767 terminal.go:GOOS() - 0s
[DEBUG] 15:51:49.767 text.go:patchTemplate:149 → {{ .Code }}
[TRACE] 15:51:49.767 terminal.go:Flags() - 0s
[DEBUG] 15:51:49.767 properties.go:GetBool:22 → always_enabled: true
[TRACE] 15:51:49.767 terminal.go:Flags() - 0s
[DEBUG] 15:51:49.767 home.go:12 → C:\Users\myusername
[TRACE] 15:51:49.767 terminal.go:GOOS() - 0s
[DEBUG] 15:51:49.767 segment.go:Execute:108 → segment: Root
[DEBUG] 15:51:49.767 text.go:patchTemplate:149 →  {{ if .Data.SSHSession }} {{ end }}{{ .UserName }}
[DEBUG] 15:51:49.767 file.go:Get:69 → cache is nil, returning empty value for key: toggle_cache
[DEBUG] 15:51:49.767 segment.go:isToggled:229 → no toggles found
[DEBUG] 15:51:49.767 properties.go:GetBool:22 → mapped_locations_enabled: true
[DEBUG] 15:51:49.767 properties.go:GetString:28 → windows_registry_icon: 
[DEBUG] 15:51:49.767 properties.go:GetString:28 → windows_registry_icon: 
[DEBUG] 15:51:49.767 home.go:12 → C:\Users\myusername
[TRACE] 15:51:49.767 terminal_windows.go:Root() - 0s
[TRACE] 15:51:49.767 separator.go:Separator() - 0s
[DEBUG] 15:51:49.767 home.go:12 → C:\Users\myusername
[TRACE] 15:51:49.767 terminal.go:GOOS() - 0s
[DEBUG] 15:51:49.767 segment.go:Execute:108 → segment: Git
[DEBUG] 15:51:49.767 file.go:Get:69 → cache is nil, returning empty value for key: toggle_cache
[TRACE] 15:51:49.767 terminal.go:GOOS() - 0s
[DEBUG] 15:51:49.767 properties.go:GetString:28 → home_icon: ~
[DEBUG] 15:51:49.767 properties.go:GetKeyValueMap:46 → mapped_locations: map[]
[TRACE] 15:51:49.767 separator.go:Separator() - 0s
[TRACE] 15:51:49.767 terminal.go:GOOS() - 0s
[TRACE] 15:51:49.767 separator.go:Separator() - 0s
[TRACE] 15:51:49.767 terminal.go:GOOS() - 0s
[TRACE] 15:51:49.767 terminal.go:GOOS() - 0s
[TRACE] 15:51:49.767 terminal.go:GOOS() - 0s
[TRACE] 15:51:49.767 terminal.go:GOOS() - 0s
[TRACE] 15:51:49.768 terminal.go:Flags() - 0s
[DEBUG] 15:51:49.768 properties.go:GetStringArray:52 → cycle: []
[DEBUG] 15:51:49.768 properties.go:GetString:28 → folder_separator_template:
[DEBUG] 15:51:49.768 properties.go:GetString:28 → folder_separator_icon: \
[DEBUG] 15:51:49.768 properties.go:GetBool:22 → cycle_folder_separator: false
[DEBUG] 15:51:49.768 properties.go:GetString:28 → folder_format: %s
[DEBUG] 15:51:49.768 properties.go:GetString:28 → edge_format: %s
[DEBUG] 15:51:49.768 properties.go:GetString:28 → left_format: %s
[DEBUG] 15:51:49.768 properties.go:GetString:28 → right_format: %s
[TRACE] 15:51:49.768 terminal.go:Flags() - 0s
[TRACE] 15:51:49.768 terminal.go:GOOS() - 0s
[TRACE] 15:51:49.768 terminal.go:StackCount() - 0s
[DEBUG] 15:51:49.768 terminal.go:ResolveSymlink:208 → C:\Users\myusername
[TRACE] 15:51:49.768 terminal.go:ResolveSymlink(C:\Users\myusername) - 1.0002ms
[DEBUG] 15:51:49.771 win32_windows.go:isWriteable:259 → not current user or in group
[DEBUG] 15:51:49.771 win32_windows.go:isWriteable:259 → not current user or in group
[DEBUG] 15:51:49.771 win32_windows.go:isWriteable:263 → current user is member of S-1-5-21-910002505-382444473-275461565-203092
[DEBUG] 15:51:49.771 win32_windows.go:isWriteable:271 ↓
    WRITE_DAC
    WRITE_OWNER
    SYNCHRONIZE
    DELETE
    READ_CONTROL
[DEBUG] 15:51:49.771 win32_windows.go:isWriteable:273 → user has write access
[TRACE] 15:51:49.771 terminal_windows.go:DirIsWritable() - 3.292ms
[DEBUG] 15:51:49.771 text.go:patchTemplate:149 →   {{ path .Data.Path .Data.Location }}
[ERROR] 15:51:49.771 terminal.go:HasParentFilePath:477 → CreateFile .git: Das System kann die angegebene Datei nicht finden.
[TRACE] 15:51:49.771 terminal.go:HasParentFilePath(.git) - 3.8158ms
[DEBUG] 15:51:49.772 text.go:patchTemplate:149 →  {{ if gt .Code 0 }}{{ else }}{{ end }}
[DEBUG] 15:51:49.772 text.go:patchTemplate:149 → {{ if gt .Code 0 }}p:red{{ end }}
[DEBUG] 15:51:49.772 text.go:patchTemplate:149 → {{ if gt .Code 0 }}p:red{{ end }}
[TRACE] 15:51:49.772 terminal.go:Shell() - 0s
[TRACE] 15:51:49.772 terminal.go:Shell() - 0s
[TRACE] 15:51:49.772 terminal.go:Flags() - 0s
[TRACE] 15:51:49.772 terminal.go:Flags() - 0s
[DEBUG] 15:51:49.772 home.go:12 → C:\Users\myusername
[TRACE] 15:51:49.772 terminal.go:GOOS() - 0s
[DEBUG] 15:51:49.772 segment.go:Execute:108 → segment: Node
[DEBUG] 15:51:49.772 file.go:Get:69 → cache is nil, returning empty value for key: toggle_cache
[DEBUG] 15:51:49.772 segment.go:isToggled:229 → no toggles found
[DEBUG] 15:51:49.772 properties.go:GetStringArray:52 → extensions: [*.js *.ts package.json .nvmrc pnpm-workspace.yaml .pnpmfile.cjs .vue]
[DEBUG] 15:51:49.772 properties.go:GetStringArray:52 → folders: []
[DEBUG] 15:51:49.772 properties.go:GetBool:22 → home_enabled: false
[DEBUG] 15:51:49.772 home.go:12 → C:\Users\myusername
[DEBUG] 15:51:49.772 home.go:12 → C:\Users\myusername
[TRACE] 15:51:49.772 terminal.go:GOOS() - 0s
[TRACE] 15:51:49.772 terminal.go:Flags() - 0s
[DEBUG] 15:51:49.772 segment.go:Execute:108 → segment: Time
[DEBUG] 15:51:49.772 home.go:12 → C:\Users\myusername
[TRACE] 15:51:49.772 terminal.go:GOOS() - 0s
[DEBUG] 15:51:49.772 file.go:Get:69 → cache is nil, returning empty value for key: toggle_cache
[DEBUG] 15:51:49.772 segment.go:Execute:108 → segment: Shell
[DEBUG] 15:51:49.772 file.go:Get:69 → cache is nil, returning empty value for key: toggle_cache
[DEBUG] 15:51:49.772 segment.go:isToggled:229 → no toggles found
[DEBUG] 15:51:49.772 segment.go:isToggled:229 → no toggles found
[DEBUG] 15:51:49.772 properties.go:GetKeyValueMap:46 → mapped_shell_names: map[]
[TRACE] 15:51:49.772 terminal.go:Shell() - 0s
[TRACE] 15:51:49.772 terminal.go:Flags() - 0s
[TRACE] 15:51:49.772 terminal.go:Flags() - 0s
[DEBUG] 15:51:49.773 home.go:12 → C:\Users\myusername
[TRACE] 15:51:49.773 terminal.go:GOOS() - 0s
[DEBUG] 15:51:49.773 segment.go:Execute:108 → segment: Go
[DEBUG] 15:51:49.773 file.go:Get:69 → cache is nil, returning empty value for key: toggle_cache
[DEBUG] 15:51:49.773 segment.go:isToggled:229 → no toggles found
[TRACE] 15:51:49.773 terminal.go:Flags() - 0s
[DEBUG] 15:51:49.773 properties.go:GetString:28 → time_format: 15:04:05
[DEBUG] 15:51:49.773 properties.go:GetStringArray:52 → extensions: [*.go go.mod go.sum go.work go.work.sum]
[DEBUG] 15:51:49.773 properties.go:GetStringArray:52 → folders: []
[DEBUG] 15:51:49.773 properties.go:GetBool:22 → home_enabled: false
[DEBUG] 15:51:49.773 home.go:12 → C:\Users\myusername
[DEBUG] 15:51:49.773 home.go:12 → C:\Users\myusername
[TRACE] 15:51:49.773 terminal.go:GOOS() - 0s
[DEBUG] 15:51:49.773 segment.go:Execute:108 → segment: Python
[DEBUG] 15:51:49.773 file.go:Get:69 → cache is nil, returning empty value for key: toggle_cache
[DEBUG] 15:51:49.773 segment.go:isToggled:229 → no toggles found
[DEBUG] 15:51:49.773 properties.go:GetString:28 → display_mode: files
[DEBUG] 15:51:49.773 properties.go:GetStringArray:52 → extensions: [*.py *.ipynb pyproject.toml venv.bak]
[DEBUG] 15:51:49.773 properties.go:GetStringArray:52 → folders: [.venv venv virtualenv venv-win pyenv-win]
[DEBUG] 15:51:49.773 properties.go:GetBool:22 → home_enabled: false
[DEBUG] 15:51:49.773 home.go:12 → C:\Users\myusername
[DEBUG] 15:51:49.774 text.go:patchTemplate:149 → in <p:blue><b>{{ .Data.Name }}</b></>
[DEBUG] 15:51:49.775 text.go:patchTemplate:149 → at <p:blue><b>{{ .Data.CurrentDate | date "15:04:05" }}</b></>
[TRACE] 15:51:49.775 terminal.go:Shell() - 0s
[TRACE] 15:51:49.775 terminal.go:Shell() - 0s
[TRACE] 15:51:49.775 terminal.go:Flags() - 0s
[TRACE] 15:51:49.775 terminal.go:Shell() - 0s
[DEBUG] 15:51:49.776 terminal_windows.go:TerminalWidth:99 → terminal width: 143
[TRACE] 15:51:49.776 terminal_windows.go:TerminalWidth() - 449.6µs
[TRACE] 15:51:49.776 path.go:Path() - 0s
[TRACE] 15:51:49.776 terminal.go:Flags() - 0s
```

--- a/src/cache/path.go
+++ b/src/cache/path.go
@@ -11,40 +11,11 @@ import (
 
 var cachePath string
 
-func realPath(filePath string) string {
-	canonicalPath, err := filepath.EvalSymlinks(filePath)
-	if err != nil {
-		// If we can't resolve symlinks, return the original path
-		return filePath
-	}
-
-	return canonicalPath
-}
-
 func Path() string {
 	defer log.Trace(time.Now())
 
 	if cachePath != "" {
 		return cachePath
-	}
-
-	returnOrBuildCachePath := func(input string) (string, bool) {
-		// validate root path
-		if _, err := os.Stat(input); err != nil {
-			return "", false
-		}
-
-		// validate oh-my-posh folder, if non existent, create it
-		cachePath := filepath.Join(input, "oh-my-posh")
-		if _, err := os.Stat(cachePath); err == nil {
-			return realPath(cachePath), true
-		}
-
-		if err := os.Mkdir(cachePath, 0o755); err != nil {
-			return "", false
-		}
-
-		return realPath(cachePath), true
 	}
 
 	var OK bool
@@ -54,13 +25,7 @@ func Path() string {
 		return cachePath
 	}
 
-	// WINDOWS cache folder, should not exist elsewhere
-	if cachePath, OK = returnOrBuildCachePath(os.Getenv("LOCALAPPDATA")); OK {
-		return cachePath
-	}
-
-	// get XDG_CACHE_HOME if present
-	if cachePath, OK = returnOrBuildCachePath(os.Getenv("XDG_CACHE_HOME")); OK {
+	if cachePath, OK = platformCachePath(); OK {
 		return cachePath
 	}
 
@@ -75,6 +40,24 @@ func Path() string {
 		return cachePath
 	}
 
-	cachePath = realPath(path.Home())
 	return cachePath
+}
+
+func returnOrBuildCachePath(input string) (string, bool) {
+	// validate root path
+	if _, err := os.Stat(input); err != nil {
+		return "", false
+	}
+
+	// validate oh-my-posh folder, if non existent, create it
+	cachePath := filepath.Join(input, "oh-my-posh")
+	if _, err := os.Stat(cachePath); err == nil {
+		return cachePath, true
+	}
+
+	if err := os.Mkdir(cachePath, 0o755); err != nil {
+		return "", false
+	}
+
+	return cachePath, true
 }

--- a/src/cache/path_unix.go
+++ b/src/cache/path_unix.go
@@ -1,0 +1,13 @@
+//go:build !windows
+
+package cache
+
+import "os"
+
+func platformCachePath() (string, bool) {
+	if cachePath, OK := returnOrBuildCachePath(os.Getenv("XDG_CACHE_HOME")); OK {
+		return cachePath, true
+	}
+
+	return "", false
+}

--- a/src/cache/path_windows.go
+++ b/src/cache/path_windows.go
@@ -1,0 +1,52 @@
+package cache
+
+import (
+	"os"
+	"path/filepath"
+	"syscall"
+	"time"
+	"unsafe"
+
+	"github.com/jandedobbeleer/oh-my-posh/src/log"
+)
+
+func platformCachePath() (string, bool) {
+	if pfn, OK := PackageFamilyName(); OK {
+		// WINDOWS MSIX cache folder, will only be present when oh-my-posh is installed via MSIX
+		msixLocalAppData := filepath.Join(os.Getenv("LOCALAPPDATA"), "Packages", pfn, "LocalCache", "Local")
+		if cachePath, OK := returnOrBuildCachePath(msixLocalAppData); OK {
+			return cachePath, true
+		}
+	}
+
+	// WINDOWS cache folder, should not exist elsewhere
+	if cachePath, OK := returnOrBuildCachePath(os.Getenv("LOCALAPPDATA")); OK {
+		return cachePath, true
+	}
+
+	return "", false
+}
+
+func PackageFamilyName() (string, bool) {
+	log.Trace(time.Now(), "PackageFamilyName")
+
+	kernel32 := syscall.NewLazyDLL("kernel32.dll")
+	procGetCurrentPackageFamilyName := kernel32.NewProc("GetCurrentPackageFamilyName")
+
+	var length uint32 = 256
+	buf := make([]uint16, length)
+	ret, _, _ := procGetCurrentPackageFamilyName.Call(
+		uintptr(unsafe.Pointer(&length)),
+		uintptr(unsafe.Pointer(&buf[0])),
+	)
+
+	if ret != 0 {
+		log.Debug("failed to get PackageFamilyName")
+		return "", false
+	}
+
+	pfn := syscall.UTF16ToString(buf)
+	log.Debug("PackageFamilyName:", pfn)
+
+	return pfn, true
+}

